library/Fission/Web/Routes.hs

```diff
@@ -24,9 +23,6 @@ type API = IPFSRoute
       :<|> AuthRoute
       :<|> PingRoute
 
-type PublicAPI = IPFSRoute
-            :<|> AuthRoute
```

Comment:
Removed because we just have `/docs` for everything now. No distinction required.